rio/src/RIO/Prelude/Logger.hs

```diff
@@ -401,15 +403,35 @@ data LogOptions = LogOptions
 --
 -- @since 0.0.0.0
 setLogMinLevel :: LogLevel -> LogOptions -> LogOptions
-setLogMinLevel level options = options { logMinLevel = level }
+setLogMinLevel level options = options { logMinLevel = return level }
+
+-- | Set the minimum log level. Messages below this level will not be
+-- printed.This modifier allows to alter the verbose format value dynamically at
```

Comment:
Needs a space between `printed.` and `This`. Also, alternative text could simply be to refer to `setLogLevelIO` and explain that it allows dynamic updates at runtime.